app/objects/secondclass/c_link.py

```diff
@@ -2,6 +2,7 @@
 from base64 import b64decode
 from datetime import datetime
 from importlib import import_module
+import uuid
```

Comment:
Nit: `import` should go above `from ... import ...` statements like the `import logging` statement at the top. This could go beneath `logging` because it's in the same group (standard library) and alphabetically ordered after.